samples/excel/range/range.yaml

```diff
@@ -6,191 +6,236 @@ name: Range basics
+            });
 
-        console.log("Done!");
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
     }
 
     async function setValues() {
-        await Excel.run(async (context) => {
-            let sheet = context.workbook.worksheets.getActiveWorksheet();
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
 
-            let data = [
-                ["Potato Chips", 10, 1.80, null],
-            ];
-            
-            let range = sheet.getRange("B5:E5");
-            range.values = data;
-            range.format.autofitColumns();
+                let data = [
+                    ["Potato Chips", 10, 1.80, null],
+                ];
+                
+                let range = sheet.getRange("B5:E5");
+                range.values = data;
+                range.format.autofitColumns();
 
-            await context.sync();
-        });
+                await context.sync();
+            });
 
-        console.log("Done!");
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
     }
 
     async function setFormula() {
-        await Excel.run(async (context) => {
-            let sheet = context.workbook.worksheets.getActiveWorksheet();
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
 
-            let range = sheet.getRange("E3");
-            range.formulas = [[ "=C3 * D3" ]];
-            range.format.autofitColumns();
+                let range = sheet.getRange("E3");
+                range.formulas = [[ "=C3 * D3" ]];
+                range.format.autofitColumns();
 
-            await context.sync();
-        });
+                await context.sync();
+            });
 
-        console.log("Done!");
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
     }
 
     async function setFormulas() {
-        await Excel.run(async (context) => {
-            let sheet = context.workbook.worksheets.getActiveWorksheet();
-
-            let data = [
-                ["Total Price"],
-                ["=C3 * D3"],
-                ["=C4 * D4"],
-                ["=C5 * D5"],
-                ["=SUM(E3:E5)"]
-            ];
-            
-            let range = sheet.getRange("E2:E6");
-            range.formulas = data;
-            range.format.autofitColumns();
-
-            await context.sync();
-        });
-
-        console.log("Done!");
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+
+                let data = [
+                    ["Total Price"],
+                    ["=C3 * D3"],
```

Comment:
Oh, I see, you have it below.  In that case I might do a different example here (like setting a formula to =SUM(namedrange) or multiplying by a constant or something...